repo/index.yaml

```diff
@@ -23,4 +23,22 @@ entries:
     urls:
     - spark/spark-1.0.0.tgz
     version: 1.0.0
-generated: 2019-06-27T03:20:02.356307221Z
+  speech-onprem:
+  - apiVersion: v1
+    appVersion: 0.1.0
+    created: 2019-06-27T16:47:49.7086141-07:00
+    description: A Helm chart for Kubernetes for all on-prem speech service components
```

Comment:
`on-prem` is not quite standard for any of the cognitive service containers. Can we please modify it to say `cognitive services speech containers`. Likewise for the tag on line 26.

Reply:
As we add more Helm charts, we'll follow a consistent convention for all cognitive services for example, `cognitive-services-textanalytics` or `cognitive-services-language-understanding`, etc. The `on-prem` designation looks like a bit of an outlier.

Reply:
How about **Cognitive Services Speech OnPremise** instead of **speech-onprem**?